en/components/general/wpf_to_angular_guide/one_way_binding.md

```diff
@@ -0,0 +1,115 @@
+---
+title: One-way data binding
+_description: WPF to Angular guide topics help you switch from WPF to Angular.
+_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular, One-way, binding
+---
+
+## One-way data binding in Angular
+
+One of the most powerful and widely used feature in WPF is data binding. It makes developers' life much easier, by synchronizing the business logic with views and vice versa, without having to write a single line of extra code. Without the power of it, WPF would be just a better-looking WindowsForms. 
+Luckily, Angular does support data binding. There are two types of data binding, one-way binding, and two-way binding. 
+
+### One-way data binding
+
+Data binding in Angular is super easy, and unlike in WPF we don't have to worry about a data context or a view model. All we have to worry about is an HTML file and a typescript file. With any data binding, the first thing you need are properties to data bind to. So let's add a property called `text` into the component class, and set it a simple value.
+In WPF, we need to set the DataContext and bind the property in XAML:
+```csharp
+public class IgniteUIClass
+{
+  public IgniteUIClass()
+  { this.Text = "IgniteUI for Angular"; }
+  
+  public string Text
+  { get; set; }
+}
+...
+public MainWindow()
+{
+  InitializeComponent();
+  this.DataContext = new IgniteUIClass();
+}
+```
+```xml
+<Label Content="{Binding Path=Text, Mode=OneWay}"></Label>
+```
+In Angular, we are directly binding a DOM property to component's property:
+```typescript
+export class SampleComponent implements OnInit {
+  text = 'IgniteUI for Angular';
+
+  constructor() { }
+  ngOnInit() {}
+}
+```
+```html
+<h2>{{ text }}</h2>
+```
+
+### Interpolation
+
+The above code, simply bind an element in the HTML which is h2, to the value of the `text` property. In this case, we are using the `interpolation` to create one-way binding. We do that by typing double curly braces, the name of the property - in our case `text`, and two closing curly braces. Another way to achieve the same result is, to create h2 tag and bind the `text` property to its innerHTML property, again using the `interpolation` syntax:
```

Comment:
1. Suggestion for: 'The above code, simply bind an element in the HTML which is h2, to the value of the `text` property.'
In the code above, we simply display text in the HTML using binding to the value of the `text` property.
2. In this case, we ~~are~~ ~~using~~ use `interpolation` to create one-way binding.
3. Another way to achieve the same result is to ... using the `interpolation` syntax again.

Reply:
@sboykova , I addressed this, you can take a look at it and resolve this conversation if you are okay.